### DISCUSSION TITLE
I want to contribute to this project!

### DISCUSSION THREAD

**Discussion by nmpereira** [CommunityFocus/CommunityFocus#24] [Show and tell]
## 👋 Welcome!
  We're building CommunityFocus, a sharable pomodoro timer that you can share with friends. And we'd like your help!
   
If you're a developer and would like to contribute, introduce yourself below in a comment. You will receive permissions to contribute to the codebase. 

We are looking for developers at any/every stage. This project is designed to help beginners to code together in a group and gain confidence in your coding and team working abilities

To get started, comment below with:
-  An introduction of yourself 👋🏼
-  The stack you already know 👨🏼‍💻 (even if you only know a little)
-  What you hope to gain from this project 💪🏼!
-  Your discord username (so we can reach you!)

After leaving a comment:
- You will receive an invitation to join the CommunityFocus GitHub organization. Accepting this invitation will grant you additional permissions, such as the ability to add/edit assignees, reviewers, labels, and more.

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

- For documentation and contribution guidelines, please refer to the [CONTRIBUTING.md](https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md) file.
- You can either view all issues in the [CommunityFocus repository](https://github.com/CommunityFocus/CommunityFocus/issues), or filter issues specifically labeled as ["help wanted"](https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).

**Comment by nmpereira** [comment]
Hi, I'm Mario. I'm from the 100Devs 2022 cohort and I've worked as a software engineer at Paystone for the past 8 months. I'm comfortable with the MERN stack, and Python. 

I hope to understand and learn
- Team management
- Reviewing PRs
- Improve on Typescript, React

Discord: mariojuanaaa#4153
Other Links: https://nmpereira.com/

**Reply by Armyant98** [reply]
Hi, I'm Anthony I've been part of the catch-up crew for a bit now. And am about half way through the classes of 100 devs. Im comfortable with html,vanilla css, and JavaScript. 

I hope to learn and grow in

- Using GitHub to make PRs
- Learn frameworks for css and js 
- How to collaborate on large projects
- Improve on skills I have already gained 

Discord: Armyant98#0001

**Reply by KhuloodHassan** [reply]
Hi, I'm Khulood or Kay (I prefer Kay) and I'm part of the 100Devs catchup crew! I'm almost at the end!! I'm comfortable with HTML, CSS, JS, Node, Express, EJS and Python.

I hope to learn
- React and Typescript

Discord: Kay#6132

**Reply by RamadanCRaji** [reply]
Hello,
I am Ramadan, and I am also a part of the 100 DEVs catchup crew. I am mid-way through the program and felt comfortable with getting my hands dirty with some challenges.

I hope to learn 
-  continue building my experience by working in a group
- using GitHub
- frameworks 
-  get more proficient in TypeScript  and styled-CSS

Discord: Fresh Prince#9748

**Reply by itisMilan** [reply]
Hy,
I am Milan, and I am part of the 100DEVS catchup crew. I am comfortable with the MERN Stack. 
I hope to learn and grow in

- Working with a team.
- Understanding the structure and growth of a project from scratch.
- To upskill myself 
- For learning new technologies

Discord:TheSky#8265

**Reply by nmpereira** [reply]
Hey @itisMilan!  Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by InstincDev** [comment]
Hi I am Chanel. I am part of 100Devs and currently on the Job Hunt. I am proficient in MERN stack and Python. I also have experience with SQL, C#, and AWS.

I hope to 

- Improve my skills
- Gain more experience working with teams
- Learn Socket.io
- Help others increase their skills

Discord: InstincDev#9789

**Comment by AhmedHHamdy** [comment]
Hi, I'm Ahmed. I am part of 100Devs and am currently halfway through 100Devs. I'm comfortable with HTML, CSS, JS

Currently Learning React & Node.js

I hope to
- Improve my skills
- Gain more experience working with teams
- How to collaborate on large projects

Discord: AhmedHamdy#2608

**Comment by IntelliJinceTech** [comment]
Hi! I'm Chris, I'm part of the catchup-crew in 100Devs and finishing up classes to go on the job hunt. I'm alright(?) with HTML, CSS, JS, Node, Express, EJS. I can also help with anything product management related (3+ years product). 

Currently learning React and Typescript. 

I hope to:

- Learn new skills (testing suites like Cypress, etc.)
- Improve on current skills
- Gain more experience working as an engineer on a team (instead of PO)
- help create a great product.

Discord: Char#8386

**Comment by jsoncodez** [comment]
Hi I'm Jason.  Current focus is C++ but general experience in html/css/php

Things I'm looking to achieve here:

Being a part of a group project to learn the processes
Learn coding

**Comment by skyspace88** [comment]
hi pals! i am drey (they/them) :-) i am part of the catchup crew currently on class 47 :D i feel pretty good with html, css, javascript, node, express, ejs, tailwind and kind of react >.>!!!!

I hope to learn:

- strengthen my react / typescript skills
- gain experience working in a group
- learn socket.io

Discord: drey#8367
other links :) https://dreymoreau.netlify.app

**Comment by XakHD** [comment]
Hello, My name is Zach. I am currently on class 17 of 100 Devs, I have a certification from John Hopkins University for Front End. I feel Comfortable with HTML,CSS,JS

I want to improve my Skills on Coding Job-related projects
I hope my knowledge from Managing construction housing projects can transfer to team based projects in a Software Engineering Environment
my goal is to become a well needed Software Engineer

Discord XakHD#4315

**Comment by eva-lng** [comment]
Hello! I'm Eva, I'm part of the catchup crew of 100 Devs, currently on class 34. I'm comfortable with HTML, CSS and JS.

Things I hope to learn:

- collaborating on larger projects and team management
- get more familiar with GitHub, learn how to make PRs
- improve my existing skills
- and maybe learn CSS and JS frameworks

Discord: noodlez#2693

**Reply by nmpereira** [reply]
Hey @eva-lng! Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by JoeWebDevelopment** [comment]
Hi, I'm Joe/Horrific 


Currently Learning HTML/CSS/JS I'm around class 16.

I'm hoping to join the project and apply directly what I am learning from the course work in #catchup-crew from 100devs, and accelerate my learning from direct application. 

Discord# Horrific#2054

**Comment by riddlew** [comment]
I'm HungryHungryHippo from the 100Devs discord and catchup crew. I've worked as a PHP and WordPress developer back in 2007-2009, and more recently as a dev intern at a telecom company for most of 2022 after a very long break from working as a dev. I'm experienced with MERN, Git,  and a little bit of Ruby on Rails.

I hope to get more experience with the Git / GitHub workflow when collaborating on a team and gain more open source contributions to add to my resume.

Discord: HungryHungryHippo#6211

**Comment by OscarViquez** [comment]
### Hi I'm Oscar HV. I am from 100Devs.
- I know: Angular, Typescript, React.js, Javascript, MySQL, Jest
- I hope to improve my skills in Unit Testing.
- Discord Name: Osh1#3021

**Comment by soph-em** [comment]
Hey I'm Sophie 👋 I've been in the catch-up crew a while now and had some coding experience prior to 100Devs. I'm from the UK so stand ups are a bit late in my time zone and I might not be able to attend all of them.

My favourite stack is Svelte/TS/Prisma/SQLite. I did a component for Together calendar (MERN), so I've had an introduction to React. Otherwise, I also like to code in Python, and occasionally dip into other technologies. 

I'm hoping to get more comfortable with a different stack and learn more about Unit Tests. 

My discord username is cactus#2550

**Reply by nmpereira** [reply]
Hey @soph-em!   Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by intelagense** [comment]
I'm Eric 🤠 and I was using the original ancestor of this timer with a study group back in early 2022. I'm cool with MERN, C/C++, and Python. 

I want to:
- Introduce more people to the Pomodoro technique.
- Help shy people get started with open-source contributions and working in teams.
- Build out a hardware desktop client device for the project. 👀

I'm "intelagense" on the [100Devs Discord](https://leonnoel.com/discord). (I'm the one always making MSPaint slides for the group events.)

**Comment by CogdornCode** [comment]
Hey I'm Cogdorn from 100Devs catchup crew. 

-I'm comfortable with HTML, CSS, and Javascript. I'm currently getting more used to Node and MongoDB.
-I want to be more comfortable w/ contributions and team projects (this would be my first!)
-Hopefully pick up some tips and tricks along the way learning from others.
-Discord: Cogdorn#3224

**Comment by HarmanRo** [comment]
Hi, I'm Harman. I am part of 100Devs and currently in catchup crew. I'm comfortable with HTML, CSS. Currently learning JS.

I want to:

-Improve my skills with JS 
-Learn about Typescript & React
-Gain experience on collaborating with teams on large projects
-Discord: Harman RO#7790

**Comment by MikeJakuszewski** [comment]
Hi, I'm Mike. I finished the 100Devs curriculum, I'm currently working on finishing the 100hours project. I'm comfortable with HTML, CSS, JS, node.js, Express, MongoDB, EJS, and learning React.

I'm hoping to:
- Get better with Git / GitHub workflow
- Learn more about React 
- Learn Typescript
- Learn socket.io
- Collaborate on an opensource project with a team

Discord: PistolPete111#1770

**Reply by nmpereira** [reply]
Hey @MikeJakuszewski!   Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by junjun107** [comment]
Hi, I'm June. I'm part of catch group of the 100Devs 2022 cohort. I'm haven been volunteering at an open source project at HackforLA for 8 months. I'm comfortable with the MERN stack. 

I hope to understand and learn
 Improve on Typescript, React. 
 Next.js
 Unit Testing
 and whatever is new and exciting! 

Discord Name: JuneBee 
Discord ID: .junebee
Linkedin: https://www.linkedin.com/in/bijun-zhao-a151586b/

**Reply by nmpereira** [reply]
Hey @junjun107!  Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by sar-mko** [comment]
Hi, I'm Sarah. I'm part of 100devs and the catchup-crew. I'm comfortable with HTML,CSS, and JS, and I'm learning node.js right now
Hoping to:
- Improve my skills with my stack 
- Gain experience contributing in an open-source project
- Learn something new
- Make something cool!

Discord: sarah, zraha#1664

**Reply by nmpereira** [reply]
Hey @sar-mko!  Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by itSnZayn** [comment]
Hi, I'm chaz apart of the 100 devs and I'm comfortable with HTML/CSS and currently learning JS rn
Hoping to:


- Improve skills
- Gain experience
- lean new ideas and how to work in a team setting
- work with cool people

Discord: popeyesbiscuitnowater

**Reply by nmpereira** [reply]
Hey @itSnZayn!  Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by Jdub679** [comment]
Hey, Jordan here, i'm part of the 100Devs catchup-crew. I'm comfortable with HTML/CSS and python. I'm also beginning to learn JS. I am looking to:

- Improve coding knowledge
- Learn how to work alongside other developers
- Help build the project

Discord: Gang#7162

**Reply by nmpereira** [reply]
Hey @Jdub679! Thank you for contributing to the app!

Please read the docs at: https://github.com/CommunityFocus/CommunityFocus/blob/main/.github/CONTRIBUTING.md
Please see and request to be assigned issues at: https://github.com/CommunityFocus/CommunityFocus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (or https://github.com/CommunityFocus/CommunityFocus/issues)

I've invited you to the CommunityFocus org. You will see an invite to the org, which will give you some additional permissions when you accept it (like adding/editing assignees, reviewers, labels etc). 

Please attend the standups on the 100Devs discord server on Tuesdays at 8pm (EST/EDT) if you want to join the discussions/standups.

**Comment by shirosensei** [comment]
Hi, I'm Tozo Sensei. I'm part of the 100Devs catchup crew. I'm currently halfway through, and I'm comfortable with HTML, CSS and JavaScript, Node, and Express.

I hope to learn

- React and Typescript
- More about Git and GitHub
- Gain experience Working with a team
- Improve my skills
- Gain pair programming experience 

Discord: tozosensei

**Comment by ec-coding** [comment]
Hey everyone, I'm Edwin. I'm a graduate of 100devs and am currently looking to contribute to community projects and work with cool people.
I'm also hoping to:
 - Practice project documentation
 - Learn TypeScript
 - Enhance my teamwork skills 
 
I go by "EC-Coding" on the 100Devs Discord

**Comment by sarafurd** [comment]
Hi, I'm Sara. I'm part of the 100Devs catchup crew. Currently on class 53. I'm comfortable with HTML, CSS, JavaScript, Node, and Express.

Hoping to:

- Gain experience by contributing to open source projects
- Improve skills
- Learn TypeScript

Discord: cozynandina#1489

**Comment by adan-ayaz-stan** [comment]
Hi, I'm Adan. I'm from the 100Devs discord group. I'm a full-stack and I'm comfortable with the React, Vue, NextJS, Nuxt, Nest and Python.

I hope to understand and learn
-     Networking
-     Building better engaging front-ends
-     To diversify my tech stack in a proper way

Discord: Spitfire | AdanAyaz#3815
Other Links: https://adan-ayaz.vercel.app

**Comment by codEno12** [comment]
Its Eno, the one who gets put on the spot. I am actually part of 100Devs, just in case someone missed it. I'm currently working on projects and freelancing. MERN is my stack, but I've played with other tech stuff and I'm willing to learn new cool tricks 

I hope to
Get comfortable with streaming
Learn new techs 
Help others along the way 
Improve my communication skills 
Discord: codeno_12

If time allows I would like to
Find new memes 
Annoy instinct

**Comment by JoseRuiz-dev** [comment]
Hi I'm Jose. I'm part of 100Devs and #catchup-crew. I am comfortable with MERN stack. 

I hope to:

-Gain experience by contributing to open source projects
-Improve my skills
-Gain more experience working with teams

Discord: jose_ruiz_dev

**Comment by codewithjazzy** [comment]
👋🏾Hi, I'm Jazzy, part of the 100Devs catchup crew and about halfway through the cohort.
👩🏾‍💻 I'm comfortable with HTML, CSS and Javascript
💪🏾Hoping to gain experience collaborating and using github, and deepening my understanding of the things I've learned so far.

My discord is: katnipgalore
